app/models/grda_warehouse/hud/client.rb

```diff
@@ -2003,11 +2003,27 @@ def race_description(include_missing_reason: false)
       HudUtility2024.race_none(self.RaceNone)
     end
 
+    def pit_gender
+      gm = gender_multi.map { |k| ::HudUtility2024.gender(k) }
+      return 'GenderNone' if gm.count.zero?
+      return 'More Than One Gender' if gm.count > 1
+
+      return HudUtility2024.gender(gm.first)
+    end
+
     def pit_race
       return 'RaceNone' if race_fields.count.zero?
-      return 'MultiRacial' if race_fields.count > 1
 
-      race_fields.first
+      race_fields_minus_latin = race_fields.reject { |x| x.include?('HispanicLatinaeo') }
+
+      return 'Hispanic/Latina/e/o (only)' if race_fields_minus_latin.count.zero?
+
+      suffix = race_fields == race_fields_minus_latin ? ' (only)' : ' & Hispanic/Latina/e/o'
+      suffix = ' (all other)' if race_fields == race_fields_minus_latin && race_fields_minus_latin.count > 1
```

Comment:
This could use a comment, but might also be unnecessary, I _think_ updating line 2024 below to just include the suffix might be sufficient.  There doesn't appear to be a multi-racial WITH latinx, but I'm not 100% sure.
```suggestion
      # if latinx wasn't one of the races, and there were more than two races, the client will be in the multi-racial category
      suffix = ' (all other)' if race_fields == race_fields_minus_latin && race_fields_minus_latin.count > 1
```